naughty/fedora-40/6621-selinux-abrt-dynamicuser

```diff
@@ -0,0 +1,2 @@
+*avc:  denied  { write } for * comm="abrt-dump-journ" name="io.systemd.*" dev="tmpfs" * scontext=system_u:system_r:abrt_dump_oops_t:s0 tcontext=system_u:object_r:systemd_userdbd_runtime_t:s0 tclass=sock_file*
```

Comment:
Typo -- this is 6**2**21, not 6**6**21

Reply:
Ah the copy pasta https://github.com/cockpit-project/bots/commit/c5b6dfdc9b204b5dad5239f01d4ed0af9fe9f4ec :)

Reply:
Also adjusted 41.

Reply:
*brown paperpag*, thanks!